src/MobileApps/MyDriving/MyDriving/ViewModel/CurrentTripViewModel.cs

```diff
@@ -194,9 +194,14 @@ public string EngineLoad
             if (IsRecording)
                 return false;
 
-            var track = Logger.Instance.TrackTime("SaveRecording");
```

Comment:
After switching from Xamarin Insights to HockeyApp, TrackTime() does not do anything. HockeyApp does not have a similar api.